simulation/traffic_simulator/src/entity/entity_base.cpp

```diff
@@ -818,5 +818,32 @@ auto EntityBase::updateTraveledDistance(const double step_time) -> double
   return traveled_distance_;
 }
 
+auto EntityBase::getDistanceToSyncronizationPosition(
+  const lanelet::Id &
+    intersection_id           // dont know about this, maybe i can get the other infomation?
+  const double stop_position  // may be if the info givven is only the intersection, perhaps needs
+                              // info about how far the stop position is from the intersection
```

Comment:
Please use [CanonicalizedLaneletPose type.](https://github.com/tier4/scenario_simulator_v2/blob/2477fad87e18b3d58baa926f779d888bc4cfa3e0/simulation/traffic_simulator/include/traffic_simulator/data_type/lanelet_pose.hpp#L26)

Reply:
And also, I suggest to rename `getDistanceToTargetLaneletPose`